CMakeLists.txt

```diff
@@ -67,6 +67,7 @@ add_subdirectory(src)
 
 if(BUILD_TESTING)
   add_subdirectory(test)
+  add_subdirectory(example)
```

Comment:
Please always build the examples, not only when testing.

Reply:
Done!